src/sock.c

```diff
@@ -109,12 +116,18 @@ sock_t sock_connect(const char *host, unsigned short port)
     return sock;
 }
 
-int sock_set_keepalive(sock_t sock, int timeout, int interval)
+int sock_set_keepalive(
+    sock_t sock, int timeout, int interval, int count, int user_timeout)
 {
     int ret;
     int optval = (timeout && interval) ? 1 : 0;
 
-    /* This function doesn't change maximum number of keepalive probes */
+#ifndef TCP_KEEPCNT
+    (void)count;
+#endif
+#ifndef TCP_USER_TIMEOUT
+    (void)user_timeout;
+#endif
```

Comment:
```suggestion
    UNUSED(count);
    UNUSED(user_timeout);
```

Removing the `ifndef` shouldn't hurt or not?

If you think it has its advantage to only mark them as `UNUSED` when the value isn't defined, please put the `UNUSED()` in the according `#else` path.